# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/shadow/pamd-userdel.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./etc/pam.d/userdel.orig	2004-12-22 20:15:47.894472968 +0100
+++ ./etc/pam.d/userdel	2004-12-22 20:15:23.889122336 +0100
@@ -0,0 +1,6 @@
+#%PAM-1.0 
+# /etc/pam.d/userdel
+auth		sufficient	pam_rootok.so
+auth		required	pam_unix.so
+account	required	pam_permit.so
+password	include		system-auth
